### PR TITLE
Update README.md with travis information

### PR DESCRIPTION
**Pull request by gardleopard**
This adds information about the travis integration to the README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,26 @@ jenkins:
...
+
+```
+travis:
+  enabled: true
+  # Travis names are prefixed with travis- inside igor.
+  masters:
+  - name: ci # This will show as travis-ci inside spinnaker.
+    baseUrl: https://travis-ci.org
+    address: https://api.travis-ci.org
+    githubToken: 6a7729bdba8c4f9abc58b175213d83f072d1d832
```
Comment (duftler) [inline README.md]: Hoping this is just random characters.

File: README.md [hunk 2]
```diff
@@ -29,6 +29,26 @@ jenkins:
...
+```
+
+Currently Travis is used within Spinnaker to trigger builds and provide artifact information for the bake stages.
```
Comment (duftler) [inline README.md]: Should this read '...to trigger _pipelines_ and provide...'?

Comment (gardleopard) [inline README.md]: Yes, you are right. Thanks!

File: README.md (hunk 1 above)
Comment (gardleopard) [inline README.md]: Yes, I was looking at the jenkins one and that has some random password as well.

**Comment by gardleopard** [comment]
@duftler s/builds/pipelines/ is done and commits squashed.

**Comment by duftler** [comment]
LGTM.

**Pull request merged**